gendiff/core.py

```diff
@@ -1,46 +1,11 @@
-from typing import Any, Dict, Set
+from typing import Dict, Any
 
 from gendiff.file_operations import parse_file
+from gendiff.diff import get_diff_view
 
 
-def get_keys_groups(
-    dict1: Dict[str, Any], dict2: Dict[str, Any]
-) -> Dict[str, Set[str]]:
-    keys1_set = set(dict1.keys())
-    keys2_set = set(dict2.keys())
-
-    union = keys1_set | keys2_set
-    common = keys1_set & keys2_set
-    missing = keys1_set - common
-
-    return {
-        "union": union,
-        "common": common,
-        "missing": missing,
-    }
-
-
-def get_diff_str(dict1: Dict[str, Any], dict2: Dict[str, Any]) -> str:
```

Comment:
[optional] I'd rather move this to `formatting` module than delete (because you'd need this code in the next step)

Reply:
consider it as an extra task - try to move it into `formatting` and have 4/5 tests from `test_diff` pass